src/index.html

```diff
@@ -6,9 +6,67 @@
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    >
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+      crossorigin="anonymous"
+    >
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap"
+      rel="stylesheet"
+    >
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a href="https://www.moyo.ua/" class="logo">
+        <img
+          src="./images/logo.png"
+          alt="moyo_logo"
+          class="logo__img"
+        >
+      </a>
+      <nav>
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a href="link" class="nav__link is-active">Apple</a>
+          </li>
+
+          <li class="nav__item">
+            <a href="link" class="nav__link">Samsung</a>
+          </li>
+
+          <li class="nav__item">
+            <a href="link" class="nav__link">Smartphones</a>
+          </li>
+
+          <li class="nav__item">
+            <a href="link" class="nav__link"
+            data-qa="hover">Laptops & Computers
+            </a>
```

Comment:
```suggestion
            <a
              href="link"
              class="nav__link"
              data-qa="hover"
            >
              Laptops & Computers
            </a>
```
Keep your attributes correctly formatted
![image](https://github.com/mate-academy/layout_moyo-header/assets/63243136/91fd8b26-c8e6-42f3-a535-84ab67eb16d4)